src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -1063,8 +1064,20 @@ def __init__(self, config):
 
     def freeze_feature_extractor(self):
         """
-        Calling this function will disable the gradient computation for the feature extractor so that its parameter
-        will not be updated during training.
+        Calling this function will disable the gradient computation for the feature encoder so that its parameter will
+        not be updated during training.
+        """
+        warnings.warn(
+            "The method `freeze_feature_extractor` is deprecated and will be removed in Transformers v5."
+            "Please use the equivalent `freeze_feature_encoder` method instead.",
+            FutureWarning,
+        )
+        self.hubert.feature_extractor._freeze_parameters()
```

Comment:
Good practice is to call the method that will replace it, so if you change code there for some reason, the code is automatically up to date.

Putting it just here once, but it's valid for all other models.

Reply:
> Thanks a lot for doing the switch.
> 
> The PR is slightly breaking without a typealias from the old `ModelTypeFeatureExtractor` to the new `ModelTypeFeatureEncoder` class, but this is tiny and for an internal class of the modeling file, so I think this is acceptable for recent models (I wouldn't say the same thing for Bert for instance!)

Yeah good point - will actually leave an alias